apps/explorer/lib/explorer/chain/celo/celo_election_rewards.ex

```diff
@@ -171,6 +230,87 @@ defmodule Explorer.Chain.CeloElectionRewards do
     }
   end
 
+  def get_epoch_rewards(account_hash_list, reward_type_list, from, to, page_number, page_size)
```

Comment:
you can pattern match on the nil values, and use an underscore if you want to keep the context + avoid unused variable warnings

```elixir
def get_epoch_rewards(account_hash_list, reward_type_list, _from = nil, to = _nil, page_number, page_size)
```